tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -423,6 +423,12 @@ iface_namingmode/test_iface_namingmode.py::TestShowQueue:
     conditions:
       - "topo_name in ['m0', 'mx']"
 
+iface_namingmode/test_iface_namingmode.py::TestConfigInterface::test_config_interface_speed:
+  skip:
+    reason: "Dynamic breakout for 40G is not supported for cisco 8101 platform"
```

Comment:
A little confused by the reason field. So is the port speed config not supported or dynamic break out not supported?